mmv1/go.mod

```diff
@@ -8,3 +8,56 @@ require (
 )
 
 require github.com/golang/glog v1.2.0
+
+require (
```

Comment:
I don't think this file should be modified.

Reply:
Yes added local changes by accident. Removed. Thanks.